flextape/proto/config.proto

```diff
@@ -32,7 +32,34 @@ message LicenseConfig {
 
   // Strategy to distribute licenses.
   oneof prioritizer {
-	FIFOPrioritizer fifo = 3;
-	EvenOwnersPrioritizer even_owners = 4;
+    FIFOPrioritizer fifo = 3;
+    EvenOwnersPrioritizer even_owners = 4;
   }
 }
+
+// General options for the entire instance
+message ServerConfig {
+  // Interval on which actions should refresh their queue position while in
+  // queue for an allocation.
+  // Default: 15s
+  uint32 queue_refresh_duration_seconds = 1;
+
+  // Interval on which actions should refresh their allocation while the action
+  // is executing.
+  // Default: 30s
+  uint32 allocation_refresh_duration_seconds = 2;
+
+  // Interval on which to clean up expired/released allocations and queue
+  // entries, and promote queued entries to allocations.
+  // Default: 1s
+  uint32 janitor_interval_seconds = 3;
```

Comment:
Why is the default 1 but we are going with 3 yet for other refresh rates we are lowering the time?

Reply:
3 is the proto tag value, not the default value or actual value here. Each field in a proto message needs a unique number.